tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,36,py,py3}, flake8
+envlist = py{27,36,37,py,py3}, flake8
```

Comment:
Could you also add this to `.travis.yml`?

Reply:
Changed `deploy` to `3.7` as well. (Same as pytest-selenium).